drop-oaetest.cql

```diff
@@ -0,0 +1,2 @@
+drop keyspace oaetest;
```

Comment:
Do we still need this with the new driver? If we do, need to make sure keyspace case matches the one in test util.js.